src/tpm.rs

```diff
@@ -617,6 +533,98 @@ fn read_file_output_path(output_path: String) -> std::io::Result<String> {
     Ok(contents)
 }
 
+/*
+ * Custom Error type for tpm execution error. It contains both error from the
+ * TPM command execution result and error within the run() function.
+ *
+ * Error Code:
+ *  - 10: error cause from the function call within the run()
```

Comment:
Unless I'm misunderstanding, this header is referring to the original custom error type (that didn't distinguish Rust and TPM errors) which has since been replaced. This should be removed and replaced with something describing the error type as it exists now.

Reply:
Thanks for catching this comment. I will update this.

Reply:
True. Early return save the world. hahaha